strawberryfields/tdm/tdmprogram.py

```diff
@@ -429,12 +424,13 @@ def compile(self, *, device=None, compiler=None):
             compiler (str, ~strawberryfields.compilers.Compiler): Compiler name or compile strategy
                 to use. If a device is specified, this overrides the compile strategy specified by
                 the hardware :class:`~.DeviceSpec`. If no compiler is passed, the default "TD2"
+                compiler is used unless the program is a TD3 program, in which case the "passive"
                 compiler is used. Currently, the only other allowed compiler is "gaussian".
```

Comment:
> Currently, the only other allowed compiler is "gaussian".

This line confuses me :thinking: Do we not allow "TD2" and now "passive" as compilers?

Reply:
Bad wording on my side. I've rewritten it now.